src/status_im/ui/components/react.cljs

```diff
@@ -1,51 +1,38 @@
 (ns status-im.ui.components.react
   (:require-macros [status-im.utils.views :as views])
-  (:require [goog.object :as object]
-            [reagent.core :as reagent]
+  (:require [reagent.core :as reagent]
             [status-im.ui.components.styles :as styles]
             [status-im.utils.utils :as utils]
             [status-im.utils.core :as utils.core]
             [status-im.utils.platform :as platform]
             [status-im.i18n :as i18n]
-            [status-im.react-native.js-dependencies :as js-dependencies]
+            ["react-native" :as react-native :refer (Keyboard)]
+            ["react-native-image-crop-picker" :default image-picker]
+            ["react-native-gesture-handler" :refer (TouchableWithoutFeedback)]
+            ["react-native-safe-area-context" :as safe-area-context
+             :refer (SafeAreaView SafeAreaProvider SafeAreaConsumer)]
             [status-im.ui.components.colors :as colors]
             [status-im.ui.components.typography :as typography]))
 
-(defn get-react-property [name]
-  (if js-dependencies/react-native
-    (or (object/get js-dependencies/react-native name) {})
-    #js {}))
-
-(defn adapt-class [class]
-  (when class
-    (reagent/adapt-react-class class)))
-
-(defn get-class [name]
-  (adapt-class (get-react-property name)))
-
-(def native-modules (.-NativeModules js-dependencies/react-native))
-(def device-event-emitter (.-DeviceEventEmitter js-dependencies/react-native))
-
-(def splash-screen (.-SplashScreen native-modules))
+(def native-modules (.-NativeModules react-native))
+(def device-event-emitter (.-DeviceEventEmitter react-native))
 
 ;; React Components
 
-(def app-registry (get-react-property "AppRegistry"))
-(def app-state (get-react-property "AppState"))
-(def view (get-class "View"))
-(def progress-bar (get-class "ProgressBarAndroid"))
-
-(def status-bar-class (when-not platform/desktop? (get-react-property "StatusBar")))
+(def app-state (.-AppState react-native))
+(def view (reagent/adapt-react-class (.-View react-native)))
+(def progress-bar (reagent/adapt-react-class (.-ProgressBarAndroid react-native)))
 
-(def scroll-view-class (get-class "ScrollView"))
-(def keyboard-avoiding-view-class (get-class "KeyboardAvoidingView"))
+(def status-bar-class (.-StatusBar react-native))
 
+(def scroll-view-class (reagent/adapt-react-class (.-ScrollView react-native)))
+(def keyboard-avoiding-view-class (reagent/adapt-react-class (.-KeyboardAvoidingView react-native)))
 
-(def text-class (get-class "Text"))
-(def text-input-class (get-class "TextInput"))
-(def image-class (get-class "Image"))
-(def picker-class (get-class "Picker"))
-(def picker-item-class (adapt-class (.-Item (get-react-property "Picker"))))
+(def text-class (reagent/adapt-react-class (.-Text react-native)))
```

Comment:
what's wrong with get-class ?

Reply:
it was just a utility function mostly used because without the shadow-cljs js interop we had to call goog/get and stuff